src/reactive-elements.js

```diff
@@ -107,11 +107,22 @@ React = typeof React === 'object' ? React : require('react');
         }
 
         return result;
-    }
+    };
 
     var parseAttributeValue = function (value) {
-        var regexp = /\{.*?\}/g;
-        var matches = value.match(regexp);
+        // Testing for json string first
+        var matches = value.match(/^\{((?:\{|\[)(?:.|[\r\n])*(?:\}|\]))}$/), result;
+
+        if(matches && matches.length > 0) {
+            try {
+                result = JSON.parse(matches[ 1 ]);
+                return result;
+            } catch (e) {
+                //Silently continuing
+            }
+        }
+
+        matches = value.match(/\{.*?\}/g);
```

Comment:
Are you missing the point of sending pointers as attributes? 

You may have {window.items} attribute and it will be interpreted as a pointer to window.items. Or JSON attribute {{data: 'name'}} and it will be interpreted as a JSON literal, but your PR allows for same attribute to be both since there is no exclusion for second and it always being executed.

What I propose is to add example named attribute types. Which will switch between all attribute types we have - JSON literal / pointer to object / string